#! /bin/sh /usr/share/dpatch/dpatch-run
## 90_i18n-fix.dpatch by Tobias Grimm <tg@e-tobi.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix problem with englisch texts containint "$" not used as
## DP: context separator.

@DPATCH@
diff -urNad vdr-plugin-burn-0.1.0~pre21~/i18n.c vdr-plugin-burn-0.1.0~pre21/i18n.c
--- vdr-plugin-burn-0.1.0~pre21~/i18n.c	2006-09-16 20:33:36.000000000 +0200
+++ vdr-plugin-burn-0.1.0~pre21/i18n.c	2006-10-24 01:04:19.000000000 +0200
@@ -210,7 +210,7 @@
 #endif
 #endif
 		},
-		{ "Total size: %1$.1f MB %2$s",
+		{ "$Total size: %1$.1f MB %2$s",
 		  "Gesamtgröße: %1$.1f MB %2$s",
 		  "",
 		  "",
@@ -782,7 +782,7 @@
 #endif
 #endif
 		},
-		{ "Job active (Writing: %1$d%%)",
+		{ "$Job active (Writing: %1$d%%)",
 		  "Auftrag aktiv (Schreibe: %1$d%%)",
 		  "",
 		  "",
@@ -1874,7 +1874,7 @@
 #endif
 #endif
 		},
-		{ "Recordings: %1$d, total size: %2$.1f MB",
+		{ "$Recordings: %1$d, total size: %2$.1f MB",
 		  "Aufzeichnungen: %1$d, Gesamtgröße: %2$.1f MB",
 		  "",
 		  "",
diff -urNad vdr-plugin-burn-0.1.0~pre21~/menuburn.c vdr-plugin-burn-0.1.0~pre21/menuburn.c
--- vdr-plugin-burn-0.1.0~pre21~/menuburn.c	2006-09-16 20:33:36.000000000 +0200
+++ vdr-plugin-burn-0.1.0~pre21/menuburn.c	2006-10-24 01:05:15.000000000 +0200
@@ -226,7 +226,7 @@
 		{
 			const recording_list& recordings( job_.get_recordings() );
 			if ( recordings.size() > 0 )
-				return str( boost::format( tr("Recordings: %1$d, total size: %2$.1f MB") )
+				return str( boost::format( tr("$Recordings: %1$d, total size: %2$.1f MB") )
 							% recordings.size() % ( double( job_.get_tracks_size() ) / MEGABYTE(1) ) );
 			else
 				return tr("No recordings marked for burning");
@@ -566,7 +566,7 @@
 
 				int percent;
 				if (manager::get_active()->get_is_burning(percent))
-					Add( new menu::text_item( str( boost::format( tr("Job active (Writing: %1$d%%)") ) % percent ) ) );
+					Add( new menu::text_item( str( boost::format( tr("$Job active (Writing: %1$d%%)") ) % percent ) ) );
 				else
 					Add(new menu::text_item(tr("Job active (Converting)")));
 
diff -urNad vdr-plugin-burn-0.1.0~pre21~/menuitems.c vdr-plugin-burn-0.1.0~pre21/menuitems.c
--- vdr-plugin-burn-0.1.0~pre21~/menuitems.c	2006-09-16 20:33:36.000000000 +0200
+++ vdr-plugin-burn-0.1.0~pre21/menuitems.c	2006-10-24 01:05:24.000000000 +0200
@@ -102,7 +102,7 @@
 
 	void size_text_item::update( bool cut_ )
 	{
-		SetText( str( boost::format( tr("Total size: %1$.1f MB %2$s") )
+		SetText( str( boost::format( tr("$Total size: %1$.1f MB %2$s") )
 						% ( double( m_job.get_tracks_size( cut_ ) ) / MEGABYTE(1) )
 						% ( m_job.get_requant_factor( cut_ ) > 1 ? tr("(would be shrinked)") : "" ) ).c_str() );
 	}
